### ISSUE TITLE
Make it clear that uploading will delete

### ISSUE THREAD

**Issue by jidanni** [zamojski/TowerCollector#149]
To be extra careful, the bottom line should say  "upload then delete" instead of just "upload". Because that is what is really going to happen.

![Screenshot_20210908-155112.jpg](https://user-images.githubusercontent.com/1230959/132469066-29cbd2de-1f95-45ff-8029-1ffa158b10d0.jpg)

Yes that makes the bottom line pretty long...

But I can't think of any other program where just pushing an "upload" button also automatically deletes the file.

**Comment by jidanni** [comment]
(In fact, I don't know if your app has any way of uploading without deleting, in case somebody  wants to do that for some reason.)

**Comment by jidanni** [comment]
Yes. It should be right on the button. Even if it is documented all over the place already.

**Comment by zamojski** [comment]
Initially Tower Collector was supposed to allow contribution only, that's why it deletes the measurements after upload. Sure, there's room for improvement.
